### DISCUSSION TITLE
Ability to merge discussions

### DISCUSSION THREAD

**Discussion by brandonkelly** [community/community#5138] [Discussions]
It should be possible to merge two discussions together, when they cover the same topic.

Example: https://github.com/github/feedback/discussions/3133 was “answered” pointing to another discussion proposing the same feature request.

It would be better if you could click a “Merge discussion” link in the right sidebar, search for the “canonical” discussion (https://github.com/github/feedback/discussions/2861), and select it. Ideally you’d then be given the option on which comments/answers from the current discussion should be included in the merge (including the OP). If the current discussion doesn’t actually add any relevant info, all comments could be deselected.

Any upvotes from the discussion should automatically be reapplied to the canonical one.

After merge, GitHub should leave a paper trail of the merge by keeping the old discussion around, showing a similar message at the top as issues get once they’ve been converted to a discussion. The discussion should automatically be locked and should stop showing up in search results.

![Mock screenshot of a discussion that had been merged into another, showing a banner message at the top that says “This discussion was merged into another discussion.”](https://user-images.githubusercontent.com/47792/129633521-7f668ed5-18a5-4af7-af66-10f72ec87635.png)

**Comment by leerob** [comment]
Meta, but this should be merged here 😄 https://github.com/github/feedback/discussions/3290

Would love to see this, would be very helpful managing the Next.js and Vercel Discussions.
